templates/data/kafka.html

```diff
@@ -0,0 +1,312 @@
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"},
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/f1cb23d7-vmware-logo.png",
+              alt="",
+              width="257",
+              height="256",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"},
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/9faa9b91-aws-logo.png",
+              alt="",
+              width="125",
+              height="256",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"},
+              ) | safe
+            }}
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="u-fixed-width">
+    <hr class="p-rule">
+    <div class="p-section">
+      <h2>What can you achieve with Kafka<sup>&reg;</sup>?</h2>
+      <p>Apache Kafka is the world's most popular open source streaming data platform.</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-start-large-4 col-9 is-split-on-medium">
+      <hr class="p-rule">
+      <div class="row">
+        <div class="col-3">
+          <h3 class="p-heading--5">Microservices architectures</h3>
+        </div>
+        <div class="col-6">
+          <p>Kafka makes for a high-performance, low latency middleware &mdash; ideal for intra-microservice communication in a modern microservices architecture, operating at scale.</p>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row">
+        <div class="col-3">
+          <p class="p-heading--5">Continuous business intelligence</p>
+        </div>
+        <div class="col-6">
+          <p>Connect, combine and transform streaming data sources to build an up-to-the-minute view of your enterprise.</p>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row">
+        <div class="col-3">
+          <p class="p-heading--5">Data hub integration</p>
+        </div>
+        <div class="col-6">
+          <p>Integrate event and machine data into a data hub for deeper processing.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="u-fixed-width">
+    <hr class="p-rule">
+    <div class="p-section">
+      <h2>How it works</h2>
+    </div>
+    <div class="p-section u-align--center">
+      <img src="https://assets.ubuntu.com/v1/57a2a600-kafka-diagram.svg" alt="Diagram: Canonical's automation for Apache Kafka handles deployment, configuration, security, scaling, high availability, upgrades, and monitoring. It runs on Ubuntu on cloud compute infrastructure or on Kubernetes" />
```

Comment:
Any issues using image template?

Reply:
Yeah the cloudinary was either failing to render or making the image blurred.